### ISSUE TITLE
Windows 11 not showing audio devices (dropdown empty)

### ISSUE THREAD

**Issue by taprootwizard** [GeekyEggo/SoundDeck#122]
I recently installed Sound Deck on a Windows 11 PC (running latest version) and no sound devices are showing up when I click on the dropdown.
I've seen this was a problem already in the past, and that it was solved in another thread, but it is currently broken again (maybe an update broke it?), at least on my end.

**Comment by GeekyEggo** [comment]
Hi @taprootwizard, can I ask what version of Windows 11 you’re running? I’m currently running 22H2, and am able to see devices. I would also suggest trying the latest [beta version](https://github.com/GeekyEggo/SoundDeck/raw/main/dist/com.geekyeggo.sounddeck.streamDeckPlugin) of Sound Deck.

**Comment by taprootwizard** [comment]
So, I've installed the beta version, even restarted the PC but nothing changes. Here are the details of the Windows 11 version, but it seems it's also 22H2:
![image](https://github.com/GeekyEggo/SoundDeck/assets/133594853/74b62fca-06e3-470b-9f5c-c151a7c669bc)
![image](https://github.com/GeekyEggo/SoundDeck/assets/133594853/6bd1a9a8-01b0-4470-9c23-4fc138ad072d)
that light bar you see under the dropdown is supposed to be the list

**Comment by GeekyEggo** [comment]
In Windows Explorer, if you navigate to:
```
%appdata%\Elgato\StreamDeck\Plugins\com.geekyeggo.sounddeck.sdPlugin\logs
```
Is there a log file? If so, could you please upload it to this issue.

**Comment by taprootwizard** [comment]
Update:
The devices are showing up now. Only things I did after installing the beta you provided me were:
Restart
Plug in a microphone

Could it be that it wouldn't list the devices if there are only Output devices? Because I wanted to use it to record system sounds, but had no microphone attached, and I guess your main goal with the plugin was to focus on mic input(?).

Thank you very much for your assistance!

**Comment by GeekyEggo** [comment]
Hmm, that’s peculiar. 😅 I had to double check the source code, but the capture device drop-down for the “Record” action should show _both_ playback and recording devices. When I’m back at my PC, I’ll have a further look.

**Comment by taprootwizard** [comment]
It does show both indeed, I might have worded my sentence wrong.

What I mean is that until I had no microphone connected to the PC, the list was not showing at all, probably because it's a desktop PC and has no built-in microphone, so it didn't have access to any item that it could have displayed under the INPUT section on the list, at least that's my guess.

TLDR: I think the list was not displaying because the PC had no Input device connected at all.

Thank you again for this awesome plugin and for your help, it's working fine now!

**Comment by GeekyEggo** [comment]
I'm glad to hear everything is working! 🥳 I'm going to close this issue as resolved, but if you have any further problems or questions, please don't hesitate to ask.
